### PR TITLE
[WIP] Handle slash backslash on config load

### PR DESCRIPTION
**Pull request by patrickpichler**
Fix for #21

### CODE REVIEW

File: pkg/util/parse_test.go [hunk 1]
```diff
@@ -57,13 +57,9 @@ const yamlTestPathSeparators = `
 pathFromLinux:
     - id: "here/dir/file.id"
     - name: "here\\dir/file.name"
```
Comment (patrickpichler) [inline pkg/util/parse_test.go:59]: @UnseenWizzard can you tell me what exactly we expect to happen here? 

Do expect the name to end up as `here/dir/file.name` or `here\\dir/file.name`?

File: go.sum [hunk 2]
```diff
@@ -13,23 +13,47 @@ github.com/kr/pty v1.1.8 h1:AkaSdXYQOWeaO3neb8EM634ahkXXe3jYbVh/F9lq+GI=
 github.com/kr/pty v1.1.8/go.mod h1:O1sed60cT9XZ5uDucP5qwvh+TE3NnUj51EiZO/lmSfw=
 github.com/kr/text v0.1.0 h1:45sCR5RtlFHMR4UwH9sdQ5TC8v0qDQCHnXt+kaKSTVE=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
+github.com/mattn/go-zglob v0.0.3 h1:6Ry4EYsScDyt5di4OI6xw1bYhOqfE5S33Z1OPy+d+To=
```
Comment (UnseenWizzard) [inline go.sum:16]: could you run `go mod tidy` please?

**Comment by patrickpichler** [comment]
@UnseenWizzard how should we continue with this?

**Comment by UnseenWizzard** [comment]
> 
> 
> @UnseenWizzard how should we continue with this?

We didn't really finish the discussion of which option to go with, but I like this one + being more explicit in which case we assume a variable to be a path.

Shall we finish the discussion on the other pr? 
Was curious about @didiladi 's opinion

File: pkg/util/parse.go [hunk 3]
```diff
@@ -74,6 +76,22 @@ func putOrGet(m map[string]map[string]string, key string) map[string]string {
...
 }
 
+func IsPath(str string) bool {
+	// all vars containing : are invalid paths
+	if strings.ContainsRune(str, ':') {
+		return false
+	}
+
+	// as or right now we only support id,name path references
+	match, err := zglob.Match("**/*.{id,name}", str)
```
[the remark this answers is not in the document]
Comment (patrickpichler) [reply pkg/util/parse.go:86]: Yes as this should match paths (including subpaths). 

`filepath.Glob` doesn't support subdirs.

Also a regex for filepaths is not trivial.

**Comment by patrickpichler** [comment]
Closed as #83 already fixes this :+1:

**Pull request closed without merging**